src/Uno.UI.DualScreen/DuoApplicationViewSpanningRect.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Views;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.Foundation;
+using Windows.Graphics.Display;
+using Windows.UI.ViewManagement;
+
+namespace Uno.UI.DualScreen
+{
+	public class DuoApplicationViewSpanningRects : IApplicationViewSpanningRects
+	{
+		Microsoft.Device.Display.ScreenHelper _helper;
+		List<Rect> _spanningRects = null;
+
+		private readonly List<Rect> EmptyList = new List<Rect>(0);
+
+		public DuoApplicationViewSpanningRects(object owner)
```

Comment:
Unused parameter?

Reply:
Yes, but it's based on what the code generation expects. At this point, the constructor has to have one parameter.